src/main/java/com/amazonaws/services/neptune/ExportPropertyGraphFromGremlinQueries.java

```diff
@@ -168,8 +168,10 @@ public void run() {
                     directories.writeResultsDirectoryPathAsMessage(target.description(), target);
 
                     queriesResource.writeResourcePathAsMessage(target);
-                    configFileResource.save(graphSchema, false);
-                    statsFileResource.save(exportStats, graphSchema);
+                    if (structuredOutput) {
```

Comment:
I'm opting to exclude the stats and config files for non-structured output. Prior to this PR, the stats and configs were blank for regular queries exports as the stats tracking and schema inference were non-functional. There is certainly an opportunity to begin outputting configs and stats following this PR, however I would like to leave that as a future followup as the current structure of these files is not designed for arbitrary queries results.

I believe it's preferable to withhold these files for now, and reintroduce them later when they updated to properly highlight stats from the queries export.